pkgs/development/python-modules/furo/default.nix

```diff
@@ -9,19 +9,33 @@
 buildPythonPackage rec {
   pname = "furo";
   version = "2022.1.2";
-  format = "flit";
+  format = "wheel";
```

Comment:
not ideal, but better than having to do the node packaging ourselves.

Reply:
Yeah, I don't know what else to do.